src/SBML.jl

```diff
@@ -12,6 +12,6 @@ include("utils.jl")
 sbml = (sym::Symbol) -> dlsym(SBML_jll.libsbml_handle, sym)
 
 export SBMLVersion,
-    readSBML, Model, UnitPart, Species, Reaction, getS, getLBs, getUBs, getOCs
+    readSBML, UnitPart, getS, getLBs, getUBs, getOCs
```

Comment:
cool. Honestly I think UnitPart can get removed too.

Reply:
think i got this